package.json

```diff
@@ -4,6 +4,7 @@
   "description": "econplayground",
   "author": "Nik Nyby",
   "license": "GPL-3.0+",
+  "type": "module",
```

Comment:
I made this change because I was getting a `ReferenceError`. Maybe that's a local versioning issue. If you aren't getting any errors on your end I can rebase and squash the last two commits.